ui/src/views/setting/ConfigurationValue.vue

```diff
@@ -295,7 +299,11 @@ export default {
         }
         this.$emit('change-config', { value: newValue })
         this.$store.dispatch('RefreshFeatures')
-        this.$message.success(`${this.$t('label.setting')} ${configrecord.name} ${this.$t('label.reset.config.value')}`)
+        var message = `${this.$t('label.setting')} ${configrecord.name} ${this.$t('label.reset.config.value')}`
+        if (configrecord.isdynamic) {
+          message += `. ${this.$t('message.setting.update.delay')}`
+        }
+        this.$message.success(message)
```

Comment:
this seems like a repeating pattern. Is there any way we can move it to a generic method?

Reply:
sure @DaanHoogland, will have a look. I hope it is okay if that is done in a separate UI-only PR